extensions/amp-link-rewriter/0.1/helper.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
nit: rename the file to something more relevant, for example `filter.js` or `scope.js`